projects/elife.yaml

```diff
@@ -833,6 +835,12 @@ observer:
                     - bus-articles--{instance}
                     - bus-metrics--{instance}
     aws-alt:
+        masterless:
```

Comment:
it's really not that confusing. I've elaborated on why `masterless` a little further down. It's not a hard and fast decision and it buys us some convenience.